### ISSUE TITLE
[Bug] Stats function crashes the tab/map after a while

### ISSUE THREAD

**Issue by crckdns** [AHAAAAAAA/PokemonGo-Map#2510]
## Expected Behavior
 I expect to run the stats for longer period of time to have some statistics.

## Current Behavior
After some time, about 10 minutes, the tab of Chrome is just crashing with "oops something went wrong"

## Possible Solution
I think ... nope, I'm not gonna think about solutions here, other report wasn't even touched. Just reporting now.

## Steps to Reproduce (for bugs)
1. open map with an area of over ~180 pokemon shown
2. open the stats slide.
3. wait some time
4. Chrome crashes the scanner tab, no logs in dev console, it says "console disconnected" like a real crash.

## Context
Stats are with that bug useless, can't use them and hold the slide open to see live population of it.

## Your Environment
* Version used: latest dev commit 0880e52
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Windows 10, Chrome.

**Comment by DoraKlikOpDora** [comment]
Is it a javascript crash? If so, try to check the javascript console for the problem. I guess it has to do with Google Maps and/or their API keys.

**Comment by MonkeyGoneWIld** [comment]
I had the same problem, Just need to get new API key

**Comment by crckdns** [comment]
No, like I said, the whole tab crashes. As long as I don't open the stats slide, I have no issues at all, also none of the other users. So it can't be a api key problem :D  the console is empty, as it disconnects/crashes.

**Comment by DoraKlikOpDora** [comment]
With the stats slide you mean the options? How long does it approximately take before it crashes?

**Comment by crckdns** [comment]
it's written above?   
about 10 minutes with ~150 pokemon shown.
Just did test around now..
with ~260 shown pokemon it crashes after ~5 minutes
with ~316 shown pokemon it crashes after ~3 minutes.

Nope, I mean the slide on the right side, the stats slide..which slides away if you click somewhere else on map.
Btw the name "stats" is misleading.  Stats (statistics) are meant to be accumulated numbers over time. But what this expansion does is just showing the visible pokemon in percentage. If they disappear, the number of pokemon in "stats" is also dicreasing, followed by change of shown percentage :) 
I thought, it would accumulate the info about appeared pokemon since the tab with map was loaded.

**Comment by crckdns** [comment]
here you go~ direct way to show...
![pogomap_bug_stats-memleak](https://cloud.githubusercontent.com/assets/787954/17255065/db3227d0-55b7-11e6-8b47-d52e8af5082e.jpg)

**Comment by crckdns** [comment]
Issue still exists, same as another reported issue
